### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by calebnelton**
More time bored in WebEx meetings .... is this really more fun to do? Right now it sure is.

### CODE REVIEW

File: src/Microsoft.AspNetCore.Razor/CodeGenerators/CSharpCodeGenerator.cs [hunk 1]
```diff
@@ -146,8 +146,7 @@ protected virtual void BuildConstructor(CSharpCodeWriter writer)
             using (writer.BuildConstructor(Context.ClassName))
             {
                 // Any constructor based logic that we need to add?
-            };
-        }
+            }        }
```
Comment (NTaylorMullen) [inline src/Microsoft.AspNetCore.Razor/CodeGenerators/CSharpCodeGenerator.cs]: old way was better.

Comment (calebnelton) [inline src/Microsoft.AspNetCore.Razor/CodeGenerators/CSharpCodeGenerator.cs]: That is just a line format issue, i'll correct that. The big item was removal of the unnecessary semicolon at the end

File: test/Microsoft.AspNetCore.Razor.Runtime.Test/Runtime/TagHelpers/TagHelperTypeResolverTest.cs [hunk 2]
```diff
@@ -13,14 +13,12 @@ namespace Microsoft.AspNetCore.Razor.Runtime.TagHelpers
 {
     public class TagHelperTypeResolverTest
     {
-        protected static readonly Type[] ValidTestableTagHelpers = new[]
-        {
+        protected static readonly Type[] ValidTestableTagHelpers = {
```
Comment (Eilon) [inline test/Microsoft.AspNetCore.Razor.Runtime.Test/Runtime/TagHelpers/TagHelperTypeResolverTest.cs]: Curly on the wrong line.

File: src/Microsoft.AspNetCore.Razor/CodeGenerators/CSharpCodeGenerator.cs (hunk 1 above)
Comment (NTaylorMullen) [inline src/Microsoft.AspNetCore.Razor/CodeGenerators/CSharpCodeGenerator.cs]: :+1:

**Comment by Eilon** [comment]
@calebnelton thanks for the PR. Can you squash everything into one commit?

I should note that we generally prefer avoiding such changes because they don't offer a significant improvement of the code, and it takes a fair bit of time from many people to work on reviewing, merging, and verifying the PR. Feel free to discuss with us through issues any additional changes you might have in mind.

**Comment by calebnelton** [comment]
@Eilon commits squashed

The next time I commit changes of this nature i'll make sure I combine them with other issues that will provide better improvements to the codebase.

**Comment by Eilon** [comment]
@calebnelton thanks!

**Comment by NTaylorMullen** [comment]
These changes have been merged. Thanks @calebnelton for your contribution!

**Pull request closed without merging**